lib/virtualGraphs.js

```diff
@@ -18,33 +18,41 @@ const listInfo = conn => {
 };
 
 const add = (conn, name, mappings, options, meta = {}) => {
-  const { db } = meta;
+  const { db, dataSource } = meta;
+  const body = {
+    name,
+    mappings,
+    options,
+    db,
+  };
+  if (dataSource) {
+    body.data_source = dataSource;
+  }
```

Comment:
It looks like we always send `db` even though it isn't require/could be `undefined`. But we don't do the same here for `data_source`. Is that because the API doesn't handle it correctly? Or is there some other reason for the different behavior?

Reply:
hm, it does appear that sending a data source is always required in the newest API, and providing the option doesn't seem to break older versions. i'll update it to match the db.